src/ucp/core/ucp_ep.c

```diff
@@ -588,6 +588,15 @@ ucs_status_t ucp_ep_init_create_wireup(ucp_ep_h ep, unsigned ep_init_flags,
     return UCS_OK;
 }
 
+static void ucp_ep_tl_bitmap_validate(ucp_ep_h ep, 
```

Comment:
seems this func used only once, so i'd revert it to be part of ucp_ep_create_to_worker_addr